internal/cli/capact/os.go

```diff
@@ -69,7 +61,7 @@ func TrustSelfSigned(status *printer.Status) error {
 
 func trustSelfSignedDarwin(tmpCertPath string) error {
 	// #nosec G204
-	cmd := exec.Command("/bin/sh", "-c", fmt.Sprintf("security verify-cert -c %s", tmpCertPath))
+	cmd := exec.Command("/bin/sh", "-c", fmt.Sprintf("security verify-cert -c %s >/dev/null", tmpCertPath))
 	cmd.Stdout = os.Stdout
```

Comment:
isn't it easier to use io.Discard instead of having `/dev/null` in command?

Reply:
ah, in this case yes. In cases when I'm using sudo no